data/posts/second-post.md

```diff
@@ -1,7 +1,5 @@
 ---
 title: Second post
-published_at: 2022-03-20
```

Comment:
we were missing just a tiny bit on the [coverage](https://app.codecov.io/gh/denoland/saaskit/blob/main/utils%2Fposts.ts) for `posts`, so i wanted to incorporate an example, which would check the fallbacks for `published_at` and `summary`.